src/test/java/org/broadinstitute/hellbender/utils/svd/SingularValueDecomposerUnitTest.java

```diff
@@ -31,11 +31,7 @@ public void testBasicFunction(final SingularValueDecomposer svder){
         final double[] expectedSingularValues = {3.0, 2.0};
 
         final SVD svd = svder.createSVD(m);
-        Assert.assertEquals(svd.getU().multiply(MatrixUtils.createRealDiagonalMatrix(svd.getSingularValues())).multiply(svd.getV().transpose()), m);
-        Assert.assertTrue(isUnitaryMatrix(svd.getV()), "V is unitary");
-        Assert.assertTrue(isUnitaryMatrix(svd.getU()), "U is unitary");
-        Assert.assertEquals(svd.getV().transpose().multiply(svd.getV()), MatrixUtils.createRealIdentityMatrix(2), "V is unitary");
-        Assert.assertEquals(svd.getU().transpose().multiply(svd.getU()), MatrixUtils.createRealIdentityMatrix(2), "U is unitary");
+        assertSVD(svd, m);
```

Comment:
Well, that's interesting.  Not exactly standard on Apache Commons' part, but nothing we can do about it.  Add a comment stating that Apache has an untraditional SVD and a still more untraditional understanding of what a unitary matrix is.